src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
+
+    public Ball getRandomBall() {
+        Color randomColor = ColorSupplier.getRandomColor();
+        int randomNumber = random.nextInt(100) + 1;
+        return new Ball(randomColor.toString(), randomNumber);
```

Comment:
You should use the `name()` method to get the String representation of the enum constant instead of `toString()`. Replace `randomColor.toString()` with `randomColor.name()`.